web/submission/views.py

```diff
@@ -548,15 +548,11 @@ def index(request, task_id=None, resubmit_hash=None):
         vpn_random = ""
 
         if routing.socks5.random_socks5 and socks5s:
-            socks5s_random = random.choice(socks5s.values()).get("name", False)
+            socks5s_random = socks5s[random.choice(list(socks5s.keys()))]
 
-        if routing.vpn.random_vpn:
-            vpn = list(vpns.values())
-            if vpn:
-                vpn_random = random.choice(vpn).get("name", False)
-
-        if socks5s:
-            socks5s_random = random.choice(list(socks5s.values())).get("name", False)
+        from lib.cuckoo.core.rooter import vpns
```

Comment:
is not good to import this over and over one page load

Reply:
ops, fixed it!